packages/graph-editor/src/components/panels/nodeSettings/index.tsx

```diff
@@ -120,11 +134,11 @@ const NodeDescription = ({
       <Textarea
         placeholder={selectedNode.factory.description}
         onChange={onChangeDesc}
-        value={annotations[description] as string}
+        value={localDescription}
       />
     </Stack>
   );
-};
+});
 
 const NodeSettings = ({
```

Comment:
We could actually do an observer here in the node Settings once to auto rerender the attributes here